src/celastro/astro.h

```diff
@@ -25,8 +25,36 @@
 namespace celestia::astro
 {
 
-constexpr inline float SOLAR_ABSMAG = 4.83f;
-constexpr inline float LN_MAG = 1.0857362f; // 5/ln(100)
+// Angle between J2000 mean equator and the ecliptic plane: 23 deg 26' 21".448
+// Seidelmann, Explanatory Supplement to the Astronomical Almanac (1992), eqn 3.222-1
+constexpr inline double J2000Obliquity    = 23.4392911_deg;
+
+// CODATA 2022
+constexpr inline double speedOfLight      = 299792.458; // km/s
+constexpr inline double G                 = 6.67430e-11; // N m^2 / kg^2
+
+// IAU 2015 Resolution B3 + CODATA 2022
+constexpr inline double SolarMass         = 1.3271244e20 / G; // kg
+constexpr inline double EarthMass         = 3.986004e14 / G; // kg
+constexpr inline double LunarMass         = 7.346e22; // kg
+constexpr inline double JupiterMass       = 1.2668653e17 / G; // kg
+
+// https://mips.as.arizona.edu/~cnaw/sun.html for Johnson V filter
+constexpr inline float SOLAR_ABSMAG       = 4.81f;
+
+// IAU 2015 Resolution B3
+constexpr inline float SOLAR_IRRADIANCE   = 1361.0f; // W / m^2
+constexpr inline float SOLAR_POWER        = 3.828e26f; // W
+
+// Bessel (1979) for Johnson V filter
+constexpr inline float VEGAN_IRRADIANCE   = 3.640e-11f; // W / m^2
+
+// Auxiliary magnitude conversion factor
+constexpr inline float LN_MAG             = 5.0f / std::log(100.0f);
```

Comment:
`std::log` is not constexpr until C++26. It works in GCC as an extension, but this is not supported on other compilers (Clang, MSVC, ...).

```suggestion
constexpr inline float LN_MAG             = 1.0857362f; // 5/ln(100)
```